CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+**Note:** This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1] - 2023-02-17
```

Comment:
🔧 These changes are test only, so there’s no need for a version bump and release. Would you back out the version bump if you haven’t released a version yet please?

Reply:
Unfortunately, I have already released the gem. Will keep it in mind for future changes.